index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="styles/main.css"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Tanya Griego</title>
+</head>
+<body>
+  <header>
+    <nav class="main-nav">
+      <div class="icon icon-menu"></div>
+      <ul>
+        <li class= "tab" data-content="home"><a href="#">Home</li>
+        <li class= "tab" data-content="projects"><a href="#">Projects</li>
+        <li class= "tab" data-content="about"><a href="#">About</li>
+        <li class= "tab" data-content="contact"><a href="#">Contact</li>
+      </ul>
+    </nav>
+    </header>
+  <section class = "tab-content" id="home">
+    <!-- <img src="images/profile.jpg" alt="profilePic"> -->
+      <h3 class="name">Tanya Griego</h3>
+      <p class="tagline">Developer in the making</p>
+      <a href="https://github.com/tanyagriego/" target="_blank" class="icon-github"></a></li>
+      <a href="https://www.linkedin.com/in/tanyagriego/" target="_blank" class="icon-linkedin"></a></li>
```

Comment:
another `</li>` to remove

Reply:
You don't have a class `icon-linkedin` in your `icon.css` that's why it's not showing up. You might need to redownload your fonts from icomoon to include the linkedin icon.